README.md

```diff
@@ -1,2 +1,89 @@
-# DumE
-Open Projects 2021
+
+
+## DumE
+
+Open Projects 2021
+![DumE](https://github.com/rodion0917/Random/blob/main/Dum-E%20in%20Action%20-%20Green%20Object.png)
+***
+
+
+## Abstract
+
+Dum-E is a pick-up and place, robotic arm. Equipped with image processing and computer vision,it can 
+detect and classify objects on the basis of their color or shape. Currently, it can operate in static
+environments, that is where obstacles and the object are not moving.
+***
+
+
+## Motivation
+
+The name Dum-E, itself is a hint of our source if inspiration for doing this project. After all,who 
+hasn't watched Iron Man. Dum-E first appeared in Iron Man I, though it keep messing things for Tony, 
+it had saved his life too. 
+Our inspiration was simply to make something cool and learn the basics of robotics, computer vision,
+image processing and the various platforms which can be used for designing and simulating our robotic arm. 
+***
+
+
+
+## Workflow
+![Workflow](https://github.com/rodion0917/Random/blob/main/Work-flow%20chart.png)
+![Picking up the object - Workflow](https://github.com/rodion0917/Random/blob/main/Picking%20up%20the%20object%20-%20Workflow.png)
+![Placing the object - Workflow](https://github.com/rodion0917/Random/blob/main/Placing%20the%20object%20-%20Workflow.png)
+***
+
+

```

Comment:
Insert a 'Software Aspect' heading and elaborate your control logic & algorithms here.